public/js/directives.js

```diff
@@ -81,7 +81,15 @@ angular.module('mean.directives', [])
       templateUrl: '/views/timer.html',
       link: function(scope, elem, attr) {}
     };
-  }).directive('landing', function() {
+  })
+  .directive('invitelist', function() {
```

Comment:
Directives should be named in camelCase. A good name for this should have been `inviteList`. And can be used in the UI as `<invite-list></invite-list>
`